src/main/java/com/blogify/blogapi/model/Post.java

```diff
@@ -0,0 +1,21 @@
+package com.blogify.blogapi.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.ToString;
+
+import java.io.Serializable;
+import java.time.Instant;
+
+@Data
+@ToString
+@Builder
+@AllArgsConstructor
+public class Post implements Serializable {
+    private String id;
+    private String title;
+    private String content;
+    private Instant creationDatetime;
```

Comment:
Add `@CreationTimestamp` to ensure that the fields value will be set to current timestamp when the entity is first saved.

Reply:
Are you sure ? if we have latence with wifi or connection problem i think it s better to handle the creation datetime from front no ? @NyH1409